client/rest/build.gradle

```diff
@@ -102,8 +102,13 @@ if (isIdea == false && isEclipse == false) {
   // removes the -nodeps jar
   configurations.runtime.artifacts.removeAll { it.archiveTask.is jar }
 
-  // removes the runtime configuration inheritance from compile
+  // runtimeElements does not exist in 3.3
+  configurations.maybeCreate('runtimeElements')
```

Comment:
Why not just check the version of gradle that is running to determine which configuration to manipulate? See `GradleVersion`.